test/literate_tabbed.litcoffee

```diff
@@ -69,13 +70,15 @@ if true
 
 	test "should ignore code blocks inside HTML", ->
 		eq executed, false
+		testsCount++
 
 ---
 
-*   A list item with a code block:
+*   A list item followed by a code block:
 
-		test "basic literate CoffeeScript parsing", ->
-			ok yes
+	test "basic literate CoffeeScript parsing", ->
```

Comment:
The technical term is "smaller attack surface". And reducing attack surface is called being responsible, especially for a high profile project like this. With tons of noob developers and mis-configured servers running node process with root-level access out there, you can never be too careful. 

This's only my opinion and it's up to you guys to decide. I won't push for it further. :)